src/CrateFactoryV1.sol

```diff
@@ -29,15 +27,12 @@ contract CrateFactoryV1 is Ownable2Step, ReentrancyGuard {
         nonReentrant
         returns (address)
     {
-        address sender = LibMulticaller.sender();
```

Comment:
Why remove multicaller?

Reply:
What was it achieving? AFAIK the only time it was invoked was to retrieve the sender, which can be done with `msg.sender` or `_msgSender()` via Context.sol if you're worried about other calls from contracts?

Reply:
iirc it was so that a user can launch a token and buy some in one transaction